qutebrowser/javascript/pac_utils.js

```diff
@@ -39,7 +39,7 @@
 
 /*
    Script for Proxy Auto Config in the new world order.
-       - Gagan Saksena 04/24/00
+       - Gagan Saksena 04/24/00 
```

Comment:
trailing space

Reply:
@imransobir said regarding `pac_utils.js`

> I think originally I didn't want to modify that because it's copied from elsewhere, but seeing that it hasn't seen a change for a long time upstream, it's probably fine to change it?

So i reset it which added the white space back in where it was oraginaly.

so this pr does not edit `pac_utils.js` not anymore

Reply:
That was me saying this :wink:

Note that I actually *prefer* cleaning it up though, as it hasn't been changed [here](https://github.com/pacparser/pacparser/blob/master/src/pac_utils.h) in a long time, and it's not even in the Firefox repo anymore - so I think it's better to keep it consistent with the rest. Sorry if that wasn't clear before!

Reply:
> I actually prefer cleaning it up though

i would also like that and i tried but `pac_utils.js` would need to be hugely rewritten, its a real mess.

this time i only changed some minor things and it still threw tons of errors so idk what to do about it.